web/src/tippyjs.js

```diff
@@ -143,6 +143,35 @@ export function initialize() {
     // below specify the target directly, elements using those should
     // not have the tippy-zulip-tooltip class.
 
+    delegate("body", {
+        target: ".mark-draft-tooltip",
+        delay: [400, 20],
+        appendTo: () => document.body,
+        onShow(instance) {
+            let content = "Select draft";
```

Comment:
Strings like this (`"Select draft"` and so on) need to be marked for translation; have a look at some of the other `onShow()` calls in the file to see how that's done with the ` $t({defaultMessage: ''});` syntax.